frontend/src/Components/board.js

```diff
@@ -1,9 +1,20 @@
 import React from 'react';
+import { Alert } from 'shards-react';
 import Row from './row';
 
-const Board = ({ grid, complete, handleKeyPress }) => {
+const Board = ({ grid, complete, displayError, handleKeyPress }) => {
   return (
     <div className='sudoku'>
+      <div className='game'>
+        <h1>SUDOKU!</h1>
+      </div>
+      {displayError ? (
```

Comment:
In the future we need to replace this with an actual Error component. Shards-react provides an alert box.